front_end/core/sdk/sdk-meta.ts

```diff
@@ -522,6 +531,60 @@ Common.Settings.registerSettingExtension({
   defaultValue: false,
 });
 
+Common.Settings.registerSettingExtension({
+  category: Common.Settings.SettingCategory.RENDERING,
+  settingName: 'showRedrawRects',
+  settingType: Common.Settings.SettingType.BOOLEAN,
+  storageType: Common.Settings.SettingStorageType.Session,
```

Comment:
Shouldn't the settings in this file be global settings, as changing it in one session affects the others?

Reply:
I am not sure, I've followed the scheme of the other settings in the file. Also, I don't think it should affect other session. If you have two inspectors opened for two views, you can have redraw rects flashing for the one and not the other ivew